Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.py

```diff
@@ -3,7 +3,7 @@
 import difflib
 
 
-def stringSimilarity(firstString: str, second_string: str, similarityThreshold: float):
+def stringSimilarity(firstString: str, secondString: str, similarityThreshold: float):
```

Comment:
Can you please change all vars from camel case to using underscore as
firstString -> first_string, etc.